packages/rancher-vsphere-csi/charts/app-readme.md

```diff
@@ -6,6 +6,15 @@ CSI allows volume plugins to be installed on Kubernetes clusters as extensions.
 
 The CSI driver for vSphere is `csi.vsphere.vmware.com`.
 
+## Support Matrix
+
+| Rancher vSphere CSI | Minimum Kubernetes Release | Maximum Kubernetes Release |
+| ------------------- | -------------------------- | -------------------------- |
+| 2.3.1               | 1.19.x                     | 1.21.x                     |
+| 2.3.0               | 1.19.x                     | 1.21.x                     |
+| 2.1.1               |                            | 1.19.x                     |
+| 2.1.000             |                            | 1.19.x                     |
```

Comment:
should we just list 1.16.x for the blank minimums?

Reply:
I don’t think we should set a minimum on the chart, but we can indicate in the docs that before 1.16 is not officially supported